src/components/PrimaryNav/PrimaryNav.test.js

```diff
@@ -13,4 +18,40 @@ describe("Primary Nav", () => {
     );
     expect(wrapper).toMatchSnapshot();
   });
+
+  it("toggles external nav menu", () => {
+    const wrapper = mount(
+      <Router>
+        <PrimaryNav />
+      </Router>
+    );
+    wrapper.find(".p-primary-nav__toggle").simulate("click");
+    expect(wrapper.find(".p-primary-nav").hasClass("ext-nav-open")).toEqual(
+      true
+    );
+    wrapper.find(".p-primary-nav__toggle").simulate("click");
+    expect(wrapper.find(".p-primary-nav").hasClass("ext-nav-open")).toEqual(
+      false
+    );
+  });
+
+  it("applies is-selected state correctly", () => {
+    delete global.window.location;
+    global.window = Object.create(window);
+    global.window.location = {
+      pathname: "/logs"
+    };
+    const wrapper = mount(
+      <Router>
+        <PrimaryNav />
+      </Router>
+    );
+    expect(
+      wrapper.find(".internal .p-list__item:last-child").hasClass("is-selected")
```

Comment:
This test isn't very resilient as if we change the order of the nav then the test could fail/pass erroneously. Probably best to instead loop through the items to make sure only a single one has `is-selected` and that its value is the label from `logs`.